app/views/user_mailer/send_password_reset_link.html.haml

```diff
@@ -0,0 +1,3 @@
+%h1 Greetings #{@user.full_name}
+%p Please click on the link below to reset your password
+= link_to "#{reset_password_url @user.token}", reset_password_url(@user.token)
```

Comment:
You don't have to interpolate the token. `link_to reset_password_url(@user.token), reset_password_url(@user.token)`